src/main/java/org/broadinstitute/hellbender/dev/pipelines/bqsr/BaseRecalibratorDataflowUtils.java

```diff
@@ -90,6 +107,33 @@ public static void ensureReferenceIsReadable(final PipelineOptions popts, String
   // ---------------------------------------------------------------------------------------------------
   // non-public methods
 
+
+  private static PCollection<BaseRecalOutput> addQuantizationInfo(SAMFileHeader readsHeader, BaseRecalibrationArgumentCollection toolArgs, PCollection<RecalibrationTables> recal) {
+    return recal.apply(ParDo
+            .named("addQuantizationInfo")
+            .of(new DoFn<RecalibrationTables, BaseRecalOutput>() {
+              @Override
+              public void processElement(ProcessContext c) {
+                RecalibrationTables rt = c.element();
+                BaseRecalibratorWorker baseRecalibratorWorker = BaseRecalibratorWorker.fromArgs(readsHeader, toolArgs);
+                baseRecalibratorWorker.onTraversalStart(null);
```

Comment:
The reason is that the original BaseRecalibrator didn't, and I wanted to keep the code as close to the original as possible. Remember that BaseRecalibrator was the very first class from GATK I worked with.

The other reason is that the constructor doesn't know whether we're going to be reading the reference, so it doesn't know what to pass to onTraversalStart.